web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -69,7 +66,7 @@ function ProjectTopSection({
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
 
-  const { creditClass, creditClassVersion, offsetGenerationMethod, sdgIris } =
+  const { creditClass, creditClassVersion, offsetGenerationMethod } =
     parseOffChainProject(offChainProject);
```

Comment:
should we remove the part related to parsing sdg iris within `parseOffChainProject`?

Reply:
The same hook is still used to create impact data so we need to keep SDG iris parsing there. On the other hand, I removed the associated SDGs code in `ProjectTopSection` [here](https://github.com/regen-network/regen-web/pull/1846/files#diff-b8a47eb57a43900bc803c9720c608dcc8d0023fc15356196bd1ef98a176ae9d7L99-L116).

Reply:
I'm not sure to get it, where else is this used?